src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsMobileWiki.java

```diff
@@ -0,0 +1,373 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.TrackingOptInDataProvider;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.TrackingOptInModal;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+
+@InBrowser(
+    browser = Browser.CHROME,
+    emulator = Emulator.GOOGLE_NEXUS_5
+)
+public class TestAdsTrackingOptInRequestsMobileWiki extends NewTestTemplate {
```

Comment:
There are no assertions in these tests. 😿 Can we change methods like e.g. `isTrackingRequestSend()` to return `boolean` and then assert it like e.g. `Assertion.assertTrue(modal.isTrackingRequestSend(foo, bar))`?

It would be much, much clearer and easier to maintain such tests.